lib/hsa/mcwamp_hsa.cpp

```diff
@@ -4509,8 +4751,6 @@ HSACopy::syncCopyExt(hc::hcCommandKind copyDir, const hc::AmPointerInfo &srcPtrI
             DBOUT(DB_COPY, "HSACopy::syncCopy(), hsa_amd_memory_async_copy() returns: 0x" << std::hex << hsa_status << std::dec <<"\n");
             throw Kalmar::runtime_exception("hsa_amd_memory_async_copy error", hsa_status);
         }
-        Kalmar::ctx.releaseSignal(signal, signalIndex);
```

Comment:
@bensander could you provide justification why these 2 lines be removed in this PR? I thought this PR is to add kernel profiling logic. but changing these 2 lines would essentially change the behavior of HSA signal reclaim process. I'm not alluding the code is wrong, but perhaps it should be made in a separate PR?